xtrlock.man

```diff
@@ -48,3 +48,18 @@ This program is distributed in the hope that it will be useful, but
 WITHOUT ANY WARRANTY; without even the implied warranty of
 MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
 General Public License for more details.
+
+This program now allows user to use their own password instead of 
+the default user password, which often is also used for other things 
+such as root access via sudo(for many desktop users, I know that!) 
+that one may not want to reveal. Using another password and share it 
+with your friend could solved this. However, due to the reason that 
+using an one character password may cause error, it is not allowed to
+use it as a custom password.
```

Comment:
What is the problem with single-character passwords (apart from them being super insecure)? I think I don't understand yet.